source/tutorial/configure-linux-iptables-firewall.txt

```diff
@@ -218,16 +209,16 @@ Provide Access For Monitoring Systems
 
    .. optional::
 
-      For config server :program:`mongod` instances running with
-      the ``shardsvr`` value for the :setting:`~sharding.clusterRole` 
+      For shard server :program:`mongod` instances running with
```

Comment:
This could be read as shard server mongod instances running with shardsvr, and hence there might be shard server mongodb instances not running with shardsvr.

same for below.